sdk/keyvault/Azure.Security.KeyVault.Administration/tests/SessionRecords/AccessControlClientLiveTests/CreateOrUpdateRoleDefinition.json

```diff
@@ -1,15 +1,15 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://heathshsmtestshsm.managedhsm.azure.net/providers/Microsoft.Authorization/roleDefinitions/b34c69ea-d07f-5c4d-a9ea-5999dcd6292d?api-version=7.2",
+      "RequestUri": "https://heathskvtesthsm.managedhsm.azure.net/providers/Microsoft.Authorization/roleDefinitions/b34c69ea-d07f-5c4d-a9ea-5999dcd6292d?api-version=7.3-preview",
```

Comment:
In JS at least our recorder infra sanitizes this (to something like https://azure_managedhsm.managedhsm.azure.net), is that possible in dotnet? [example](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-admin/recordings/node/keyvaultaccesscontrolclient_role_definitions/recording_can_list_role_definitions.js#L141)

It's not harmful in this case, but adds noise to the recording diffs - not sure if there's similar options for dotnet?

Reply:
We don't sanitize these, no. There's already plenty other changes to recordings, so this is just noise among noise. :)